Cargo.toml

```diff
@@ -13,7 +13,7 @@ stellar-registry = { path = "crates/stellar-registry" }
 loam-sdk = { git = "https://github.com/loambuild/loam", version = "0.6.16" }
 loam-build = { git = "https://github.com/loambuild/loam", version = "0.7.3" }
 loam-subcontract-core = { git = "https://github.com/loambuild/loam", version = "0.7.9" }
-stellar-cli = { git = "https://github.com/stellar/stellar-cli", package = "soroban-cli" } 
+stellar-cli = { version = "22.8.1", package = "soroban-cli" } 
```

Comment:
Do we need `package`?

Reply:
I added it when working with it locally and we might need it for git dependency but I don’t think we need it for the published version